### ISSUE TITLE
Cannot bootstrap colcon using powershell on windows 10

### ISSUE THREAD

**Issue by sloretz** [colcon/colcon-powershell#5]
I'm unable to bootstrap colcon using only powershell on windows 10. After the first invocation of colcon the `prefix.ps1` script does not exist.

To reproduce, save this script as `bootstrap.ps1` in an empty directory and call it from powershell.

```powershell
Invoke-WebRequest -Uri https://raw.githubusercontent.com/colcon/colcon.readthedocs.org/master/colcon.repos -OutFile colcon.repos
python -m venv env
. env\Scripts\activate.ps1
pip install -U EmPy setuptools PyYAML pywin32 coloredlogs
mkdir src
vcs import src --input colcon.repos
echo $null > src/colcon-argcomplete/COLCON_IGNORE
python src/colcon-core/bin/colcon build --paths src/*
# Cannot source prefix.ps1 here because it doesn't exist
. "install/prefix.ps1"
colcon build
```

In `install` there is a `prefix.bat` script,  but no powershell script.

**Comment by dirk-thomas** [comment]
That is expected behavior. In the first build only functionality implemented in the `colcon-core` package is available (which is support for `sh` and `bat`). Only after sourcing the result of the first build and building the workspace a second time features contributed by other packages is available.

After the second pass the workspace should have additional scripts (`bash`, `ps1`, `zsh` depending on the platform).

**Comment by sloretz** [comment]
Ah that makes sense. If it's only possible to bootstrap colcon using shells that can make use of scripts provided by `colcon-core`, then it will never be possible to bootstrap using powershell right? I'll close this as `wontfix` then.

**Comment by dirk-thomas** [comment]
> then it will never be possible to bootstrap using powershell right?

Correct.
